app/javascript/react/components/ReviewTile.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const ReviewTile = (props) => {
+    return (
+        <div>
+            Rating: {props.rating} 
+            Review: {props.body}
+        </div>
```

Comment:
you may want to double check that this is formatting correctly with a single `div`, both pieces of information may appear on the same line on the page as opposed to how we see it here separated